llpc/context/llpcPipelineContext.cpp

```diff
@@ -610,13 +610,12 @@ ShaderOptions PipelineContext::computeShaderOptions(const PipelineShaderInfo &sh
     }
   }
 
-  if (ScalarizeWaterfallDescriptorLoads.getNumOccurrences() > 0) {
+  if (ScalarizeWaterfallDescriptorLoads.getNumOccurrences() > 0)
     shaderOptions.scalarizeWaterfallLoads = ScalarizeWaterfallDescriptorLoads;
-  } else {
-    shaderOptions.scalarizeWaterfallLoads = shaderInfo.options.scalarizeWaterfallLoads;
-    // Enable waterfall load scalarization when vgpr limit is set.
-    if (shaderOptions.vgprLimit != 0 && shaderOptions.vgprLimit != UINT_MAX)
-      shaderOptions.scalarizeWaterfallLoads = true;
+  else {
```

Comment:
We should usually put { } around both sides of an if/else if one of them needs it.

Reply:
Done